docs/source/zh/installation.md

```diff
@@ -0,0 +1,256 @@
+<!---
+Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 安装
+
+为你正在使用的深度学习框架安装 🤗 Transformers、设置缓存、并选择性配置 🤗 Transformers 以离线运行。
+
+🤗 Transformers 已在Python 3.6+、PyTorch 1.1.0+、TensorFlow 2.0+以及Flax上进行测试。针对你使用的深度学习框架，请参照以下安装说明进行安装：
+
+* [PyTorch](https://pytorch.org/get-started/locally/)安装说明。
+* [TensorFlow 2.0](https://www.tensorflow.org/install/pip)安装说明。
+* [Flax](https://flax.readthedocs.io/en/latest/)安装说明。
+
+## 使用pip安装
+
+你应该使用[虚拟环境](https://docs.python.org/3/library/venv.html)安装 🤗 Transformers。如果你不熟悉Python虚拟环境，请查看此[教程](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/)。使用虚拟环境，你可以轻松管理不同项目，避免不同依赖项之间的兼容性问题。
+
+首先，在项目目录中创建虚拟环境：
+
+```bash
+python -m venv .env
+```
+
+在Linux和MacOs系统中激活虚拟环境：
+
+```bash
+source .env/bin/activate
+```
+在Windows系统中激活虚拟环境：
+
+```bash
+.env/Scripts/activate
+```
+
+现在你可以使用以下命令安装 🤗 Transformers：
+
+```bash
+pip install transformers
+```
+
+若仅需CPU支持，可以使用单行命令方便地安装 🤗 Transformers 和深度学习库。例如，使用以下命令安装 🤗 Transformers 和PyTorch：
```

Comment:
Inconsistent spacing between CJK and ASCII characters. 

Generally in web typesetting environments, it may be more comfortable to read by inserting spaces between those CJK and ASCII characters. Microsoft Word do it automatically, but web browsers don't.

There are several other instances of this issues in other part of the docs.

Reply:
Thanks for your review @wfjsw, would you like to open a PR to fix this issue?

Reply:
@stevhliu Sure! I'll do it later after finish reviewing other files. Meanwhile could you take a look at #26603?

Reply:
> Inconsistent spacing between CJK and ASCII characters.
> 
> Generally in web typesetting environments, it may be more comfortable to read by inserting spaces between those CJK and ASCII characters. Microsoft Word do it automatically, but web browsers don't.
> 
> There are several other instances of this issues in other part of the docs.

The justification for the space inconsistency is that  `🤗 Transformers` seems like a proper noun so I followed the format in [快速上手](https://huggingface.co/docs/transformers/v4.34.0/zh/quicktour). On a second review, neither `🤗 Transformers简介` nor `快速上手` section has a consistent translation format. Apart from the inconsistent space problem, they use a mix of Chinese and English punctuations (mostly English punctuations but Chinese punctuations would be more appropriate in my view). If possible, please also take into account these issues while fixing.